website/docs/about/team.md

```diff
@@ -6,18 +6,18 @@ description: ORCA Team Listing
 ## Core ORCA Team Members
 
 | Role                         | Member Name    | GitHub Profile                                 |
-| ---------------------------- | -------------- | ---------------------------------------------- |
+|------------------------------|----------------|------------------------------------------------|
 | Product Owner / Scrum Master | Bradley Hazuka | [@bhazuka](https://github.com/bhazuka)         |
 | System Engineer              | Scott Saxon    | [@Quarriers89](https://github.com/Quarriers89) |
-| Lead Developer               | Jacob Campbell | [@ifestus](https://github.com/ifestus)        |
+| Lead Developer               | Jacob Campbell | [@ifestus](https://github.com/ifestus)         |
```

Comment:
Should change this to Andrew Dorn as Lead Dev and add Rizbi in as developer

Reply:
Updated